nextcloud_update.sh

```diff
@@ -35,7 +35,9 @@ if does_snapshot_exist "NcVM-snapshot-pending"
 then
     msg_box "Cannot proceed with the update currently because NcVM-snapshot-pending exists.\n
 It is possible that a backup is currently running.\n
-Advice: don't restart your system now if that is the case!"
+Advice: don't restart your system now if that is the case!\n
+If you are sure that no backup is currently running, you can fix this by executing:
+'sudo lvrename /dev/ubuntu-vg/NcVM-snapshot-pending /dev/ubuntu-vg/NcVM-snapshot'"
```

Comment:
If the updates are run with automation, the end user will never see this message since `msg_box`es gets smashed since no TERM is set.

Reply:
Yes I know. I just wanted to add this hint if someone is running the update script manually...